### ISSUE TITLE
`body` not always redundant

### ISSUE THREAD

**Issue by csswizardry** [macbre/analyze-css#49]
Hey!

First up, great tool! I’m loving that people are starting to build this kind of thing for CSS.

I just gave it a little run over some CSS I’m working on and the following was a false positive:

```
"redundantBodySelectors": [
  ".has-modal > body"
]
```

This is actually exactly what I need to have: when a modal is shown, a class of `.has-modal` is applied to the `html` element, and the `body`’s overflow etc. is configured accordingly.

I understand that the `body` in something like `body h1 {}` would be redundant, but perhaps a simple adjustment to the rule should be that if the `body` element is the _key_ selector, then it doesn’t get flagged…?
